static/portland.svg

```diff
@@ -0,0 +1,37 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="140" height="70" viewBox="0 0 140 70">
+  <style type="text/css">
+    .st0{fill:#f2f5f7;}
+    .st1{fill:#738da0;}
+    .st2{fill:#158466;}
+    .st3{fill:#2cdd90;}
+    .st4{fill: none; stroke: #f2f5f7; stroke-width: 2.4; stroke-miterlimit: 3.2;}
+    .st5{fill: none; stroke: #f2f5f7; stroke-width: 2.4; stroke-miterlimit: 2.4;}
```

Comment:
We should be exporting with these set to attributes so these classes don't inadvertently litter the DOM or complicate the CSS include.